pyalgotrading/algobulls/connection.py

```diff
@@ -489,9 +518,10 @@ def start_job(self, strategy=None, start=None, end=None, instruments=None, lots=
         self.api.set_strategy_config(strategy_code=strategy, strategy_config=strategy_config, trading_type=trading_type)
 
         # Submit trading job
-        response = self.api.start_strategy_algotrading(strategy_code=strategy, start_timestamp=start, end_timestamp=end, trading_type=trading_type, lots=lots, initial_funds_virtual=initial_funds_virtual, broker_details=broking_details)
+        response = self.api.start_strategy_algotrading(strategy_code=strategy, start_timestamp=start, end_timestamp=end, trading_type=trading_type,
+                                                       lots=lots, initial_funds_virtual=initial_funds_virtual, broker_details=broking_details, location=location)
 
-        return response
+        return response, location
```

Comment:
avoid giving back location; try to handle it in this function itself

Reply:
done